lib/puppet/provider/vcsrepo/git.rb

```diff
@@ -493,7 +535,7 @@ def tags
   def set_excludes
     # Excludes may be an Array or a String.
     at_path do
-      open('.git/info/exclude', 'w') do |f|
+      File.open(File.join(@resource.value(:path), '.git/info/exclude'), 'w') do |f|
```

Comment:
Was there an issue with the relative path?

Reply:
I think this may have been a mistake, me getting a little too aggressive finding/replacing :-p I will fix that and revert this bit to its original state